vcpkg/build_iroha_deps.sh

```diff
@@ -9,4 +48,4 @@ git -C $vcpkg_path checkout $(cat "$iroha_vcpkg_path"/VCPKG_COMMIT_SHA)
 for i in "$iroha_vcpkg_path"/patches/*.patch; do git -C $vcpkg_path apply --ignore-whitespace $i; done;
 $vcpkg_path/bootstrap-vcpkg.sh
 cat "$iroha_vcpkg_path"/VCPKG_DEPS_LIST | xargs $vcpkg_path/vcpkg install
-cat "$iroha_vcpkg_path"/VCPKG_HEAD_DEPS_LIST | xargs $vcpkg_path/vcpkg install --head
+cat "$iroha_vcpkg_path"/VCPKG_HEAD_DEPS_LIST | xargs $vcpkg_path/vcpkg install --head
```

Comment:
No newline at the end of file. Should be.
```suggestion
cat "$iroha_vcpkg_path"/VCPKG_HEAD_DEPS_LIST | xargs $vcpkg_path/vcpkg install --head

```